h2o-dist/index.html

```diff
@@ -636,6 +665,89 @@ <h2>Gradle-style specification for Maven artifacts</h2>
 &nbsp;compile "ai.h2o:h2o-web:${h2oProjectVersion}"</br>
 }</br>
           </div>
+            <div id="quickstart-kubernetes" style="display:none">
+                <h2>Get started with H<sub>2</sub>O on Kubernetes</h2>
+                <p>1. Set-up kubernetes cluster and kubectl.</p>
+                <p>2. Replace the placehorders in the YAML file above with desired values</p>
+    
+                <button class="btn copy_button" id="btnCopy7" data-copied-hint="Copied!" data-clipboard-target='to_copy7'>
+                    <span class="octicon octicon-clippy"></span>
+                </button>
+                <p class="terminal" id="to_copy7">
+apiVersion: apps/v1<br/>
```

Comment:
This is great, the other how examples are however copy&paste for the most trivial 1 node case.

Question: if we assumed a single node and hard-coded other defaults (like memory, cpu), could we provide a simple copy&paste example? Would the DNS config still be needed?

Just a question at this point ;)

Reply:
Yes, this is probably the best solution. This can be done. Definitely. Will adjust.

Reply:
Done.